ansible/provision-private-proxied-cluster.yml

```diff
@@ -0,0 +1,51 @@
+# Creates a TLS enabled cluster with tiered storage using a local CA in tls/ca for testing purposes
+# More information about tiered storage available here: https://docs.redpanda.com/docs/manage/tiered-storage/
+# !!! WARNING !!!
+# If you intend to use this for a production cluster you will need to set handle_certs to false and provide your own certs
+# Using our demo certificate authority for production is not recommended, and is intended only for testing and proof of concepts.
+# Redpanda encourages you to integrate with your corporate certificate management setup.
+# variables
+# root_ca_dir: location of our demo cert authority
+# handle_certs: when false, disables all activity related to our demo CA
+# enable_tls: when true, adds the tls template to the /etc/redpanda/redpanda.yml file
+# advertise_public_ips : causes the cluster to set a public ip on advertised_kafka_api, allowing access to the cluster from the public internet
+---
+- name: Provision client
+  hosts: client
+  vars:
+    squid_acl_localnet:
+      - "10.0.1.0/24"
```

Comment:
we should hint to the user on the squid_acl_localnet that they can change to whatever cidr range required and that the redpanda broker nodes would need to be within this range to use the proxy.

Reply:
(and wherever else we do this with squid_acl_localnet)

Reply:
I see we hardcode the tf to use this range, so a note pointing them to the TF for that as well.

Reply:
@hcoyote  the TF is not hardcoded to use this range. Hard coding in this instance would mean that we set the subnet range inside the module without allowing the user to specify it. In this instance we are setting the range to an arbitrary but standard private IP value in an example, then using it as the starting point for the playbook. The end user could specify different values if they so chose.